src/tools/pub/pubspec.md

```diff
@@ -57,8 +57,14 @@ For more information, see
 [Pub Assets and Transformers](/tools/pub/assets-and-transformers) and
 [Configuring the Built-in dart2js Compiler]({{site.webdev}}/tools/pub/dart2js-transformer).
 
-All other fields are ignored. A simple but complete pubspec looks something
-like the following:
+All other fields are ignored. 
+
+**Note:** The pubspec for [Flutter apps](https://flutter.io) supports
+[a few additional fields](https://flutter.io/assets-and-images/) for managing assets. 
```

Comment:
Replace `https://flutter.io` by `{{site.flutter}}`. Also, I'd suggest this wording:

    **Note:** [Flutter app]({{site.flutter}}) pubspecs support [a few additional fields]({{site.flutter}}/assets-and-images) for managing assets.